### ISSUE TITLE
Issues with image data update in `IMAGES` panel

### ISSUE THREAD

**Issue by joonjeon** [tensorflow/tensorboard#5293]
Hi! I am currently using Tensorboard and have encountered an issue.
The description is as follows:

#### Diagnostics Output:
``````
--- check: autoidentify
INFO: diagnose_tensorboard.py version e43767ef2b648d0d5d57c00f38ccbd38390e38da

--- check: general
INFO: sys.version_info: sys.version_info(major=3, minor=8, micro=5, releaselevel='final', serial=0)
INFO: os.name: nt
INFO: os.uname(): N/A
INFO: sys.getwindowsversion(): sys.getwindowsversion(major=10, minor=0, build=19042, platform=2, service_pack='')

--- check: package_management
INFO: has conda-meta: True
INFO: $VIRTUAL_ENV: None

--- check: installed_packages
INFO: installed: tensorboard==2.5.0
INFO: installed: tensorflow==2.5.0
INFO: installed: tensorflow-estimator==2.5.0
INFO: installed: tensorboard-data-server==0.6.1

--- check: tensorboard_python_version
INFO: tensorboard.version.VERSION: '2.5.0'

--- check: tensorflow_python_version
2021-09-04 14:06:03.149092: I tensorflow/stream_executor/platform/default/dso_loader.cc:53] Successfully opened dynamic library cudart64_110.dll
INFO: tensorflow.__version__: '2.5.0'
INFO: tensorflow.__git_version__: 'v2.5.0-rc3-213-ga4dfb8d1a71'

--- check: tensorboard_data_server_version
INFO: data server binary: None

--- check: tensorboard_binary_path
INFO: which tensorboard: b'C:\\Users\\joonjeon\\miniconda3\\Scripts\\tensorboard.exe\r\n'

--- check: addrinfos
socket.has_ipv6 = True
socket.AF_UNSPEC = <AddressFamily.AF_UNSPEC: 0>
socket.SOCK_STREAM = <SocketKind.SOCK_STREAM: 1>
socket.AI_ADDRCONFIG = <AddressInfo.AI_ADDRCONFIG: 1024>
socket.AI_PASSIVE = <AddressInfo.AI_PASSIVE: 1>
Loopback flags: <AddressInfo.AI_ADDRCONFIG: 1024>
Loopback infos: [(<AddressFamily.AF_INET6: 23>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('::1', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('127.0.0.1', 0))]
Wildcard flags: <AddressInfo.AI_PASSIVE: 1>
Wildcard infos: [(<AddressFamily.AF_INET6: 23>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('::', 0, 0, 0)), (<AddressFamily.AF_INET: 2>, <SocketKind.SOCK_STREAM: 1>, 0, '', ('0.0.0.0', 0))]

--- check: readable_fqdn
INFO: socket.getfqdn(): 'DESKTOP-JOONJEON'

--- check: stat_tensorboardinfo
INFO: directory: C:\Users\joonjeon\AppData\Local\Temp\.tensorboard-info
INFO: os.stat(...): os.stat_result(st_mode=16895, st_ino=16325548649238242, st_dev=2456407384, st_nlink=1, st_uid=0, st_gid=0, st_size=0, st_atime=1630731956, st_mtime=1630731956, st_ctime=1627466907)
INFO: mode: 0o40777

--- check: source_trees_without_genfiles
INFO: tensorboard_roots (1): ['C:\\Users\\joonjeon\\miniconda3\\lib\\site-packages']; bad_roots (0): []

--- check: full_pip_freeze
INFO: pip freeze --all:
absl-py==0.13.0
astunparse==1.6.3
attrs==21.2.0
brotlipy==0.7.0
cachetools==4.2.2
certifi==2020.6.20
cffi @ file:///C:/ci/cffi_1605538112425/work
chardet @ file:///C:/ci/chardet_1605303225733/work
conda==4.9.2
conda-package-handling @ file:///C:/ci/conda-package-handling_1603003327818/work
cryptography @ file:///C:/ci/cryptography_1605526388508/work
cycler==0.10.0
dill==0.3.4
flatbuffers==1.12
future==0.18.2
gast==0.4.0
google-auth==1.32.1
google-auth-oauthlib==0.4.4
google-pasta==0.2.0
googleapis-common-protos==1.53.0
grpcio==1.34.1
h5py==3.1.0
idna @ file:///tmp/build/80754af9/idna_1593446292537/work
importlib-resources==5.2.0
keras-nightly==2.5.0.dev2021032900
Keras-Preprocessing==1.1.2
kiwisolver==1.3.1
lxml==4.6.3
Markdown==3.3.4
matplotlib==3.4.2
menuinst==1.4.16
numpy==1.19.5
oauthlib==3.1.1
opencv-python==4.5.3.56
opt-einsum==3.3.0
Pillow==8.2.0
pip==20.2.4
promise==2.3
protobuf==3.17.3
pyasn1==0.4.8
pyasn1-modules==0.2.8
pycosat==0.6.3
pycparser @ file:///tmp/build/80754af9/pycparser_1594388511720/work
pyOpenSSL @ file:///tmp/build/80754af9/pyopenssl_1605545627475/work
pyparsing==2.4.7
PySocks @ file:///C:/ci/pysocks_1605287845585/work
python-dateutil==2.8.1
pywin32==227
PyYAML==5.4.1
requests @ file:///tmp/build/80754af9/requests_1592841827918/work
requests-oauthlib==1.3.0
rsa==4.7.2
ruamel-yaml==0.15.87
scipy==1.6.3
setuptools==50.3.1.post20201107
six @ file:///C:/ci/six_1605187374963/work
tensorboard==2.5.0
tensorboard-data-server==0.6.1
tensorboard-plugin-wit==1.8.0
tensorflow==2.5.0
tensorflow-datasets==4.3.0
tensorflow-estimator==2.5.0
tensorflow-metadata==1.1.0
termcolor==1.1.0
tqdm @ file:///tmp/build/80754af9/tqdm_1605303662894/work
typing-extensions==3.7.4.3
urllib3 @ file:///tmp/build/80754af9/urllib3_1603305693037/work
Werkzeug==2.0.1
wheel==0.35.1
win-inet-pton @ file:///C:/ci/win_inet_pton_1605306167264/work
wincertstore==0.2
wrapt==1.12.1
zipp==3.5.0

``````

## Issue description

I have built some object detection model (e.g. Faster R-CNN).
Then, I tried to check if it the detection output images are being visualized properly.
The detected bounding boxes are drawn on the test set images using `pillow` module,
and the images were to be displayed on the `IMAGES` panel using `tf.summary.image` function.
However, after first epoch of training and testing, even after numerous epochs of training,
**no further update is being applied to the images shown on the `IMAGES` panel.**

Note that this issue is still being reproduced when using higher versions of Tensorflow and Tensorboard (e.g. 2.6.0),
and the issue persists regardless of hardware specifications of the computer I am using.
Meanwhile, **the issue is NOT reproduced when using Ubuntu 18.04 and Firefox Web browser.**

Is there a way I could fix this issue? Or is this an ongoing official issue of Tensorboard?

Please pardon me if this is a duplicate posting of the issue previously mentioned.
If there are any other necessary information, please let me know.

Thanks in advance.

**Comment by joonjeon** [comment]
Seems like more than 7 months have passed...

Any updates on the fix for this issue?

**Comment by nfelt** [comment]
I'm not aware of other users with this problem. To debug this further I think we would need a way to reproduce the issue you're experiencing, e.g. a minimal example of how you're writing data to TensorBoard and the command line you're using to launch it.

It would also be useful if you could elaborate more on a few aspects:

- It is surprising that this behavior would differ between browsers. Could you try to isolate whether the problem is with the browser that is used to _view_ TensorBoard, or with the OS used to _run_ tensorboard?  E.g. run the same TensorBoard instance with `--bind_all`, then try accessing that hostname and port from the different browsers.

- When you say "no further update is being applied to the images shown on the IMAGES panel." - are you manually clicking the reload button? Or do you have auto-reload enabled, in the settings menu under the gear icon? If you have different auto-reload settings in different browsers that could explain the browser-specific difference.

**Comment by joonjeon** [comment]
Hi there!

The issue does not seem to depend on the browser being used.
I have run the same code in Windows and executed Tensorboard to view on Firefox browser, but the issue persists. On the other hand, the issue is not reproduced when using Chrome browser in Ubuntu to view Tensorboard.

I tried to manually click the reload button regardless of the browser being used. Here the issue persists in Windows while it is not reproduced in Ubuntu.


Meanwhile, I just found out that the issue is reproduced in Windows only if I set **same values for `step` argument** when calling `tf.summary.image` throughout whole training session. Strangely enough, when I set different values for `step` argument, the issue is not reproduced in Windows, either.

Regarding this situation, could you figure out what is the problem?
Thanks in advance.
